hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/AccountStore.java

```diff
@@ -116,4 +142,12 @@ private Long getAccountNum(final AccountID id) {
         }
         return id.getAccountNum();
     }
+
+    private KeyOrLookupFailureReason validateKey(final JKey key) {
+        if (key == null || key.isEmpty()) {
```

Comment:
In current implementation keys are never `null`, so I'd lean to throwing `IllegalArgumenException` there.

Reply:
Fixed. Thanks !